index.html

```diff
@@ -52,11 +65,15 @@ <h3>I'am a Software Developer</h3>
   <section class="work">
     <div class="work-title">
       <h4>My Recent Works</h4>
+      <hr>
     </div>
     <section class="grid">
       <div class="collection">
         <div class="work1">
+          <div>
           <img src="items/img-for-first-project.png" alt="">
+        </div>
+        <div class="work1-second-part">
```

Comment:
[ ] kindly add a hover effect to your cards, so when we hover on with the mouse the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.👀
Your cards : 
![hoverta3ou](https://user-images.githubusercontent.com/62156607/193286555-f0c7a1dc-e271-4483-8be6-4094540de65c.PNG)

Figma design : 
![hoverdesign](https://user-images.githubusercontent.com/62156607/193286605-21a18138-c26e-4de6-99a0-1da72c586ec8.PNG)